crates/subspace-solving/src/codec.rs

```diff
@@ -73,6 +73,7 @@ fn mix_public_key_hash_with_piece_index(public_key_hash: &mut [u8], piece_index:
         });
 }
 
+// TODO: This is a dummy encoder now, but will likely go away in the future or will be replaced
```

Comment:
Hmm, is it okay to merge dummy code, though?

Reply:
The alternative is to update sloth256-189 library to support arbitrary sizes, which is a waste of time because we'll likely not use the library at all. So I decided just making it compile that way for now is acceptable.